src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpEncapsulateField.cs

```diff
@@ -44,6 +44,7 @@ public void EncapsulateThroughCommand()
             encapsulateField.Invoke();
             dialog.VerifyOpen(encapsulateField.DialogName);
             dialog.ClickApply(encapsulateField.DialogName);
+            VisualStudio.Workspace.WaitForAllAsyncOperations();
```

Comment:
What async operations are happening here?  

Generally our philosophy has been that some verify helper should do the appropriate waiting, instead of needing to sprinkle it through tests.